protocolspec/spec.print.txt

```diff
@@ -16,55 +16,49 @@ Command PrintHUDMessage
 	String message
 	Float x
 	Float y
-	Short hudWidth
-	Short hudHeight
+	Long type
```

Comment:
Do we really need a `Long` here? Shouldn't a `Short` be sufficient if you move `HUDMESSAGE_SEND_ALPHA` and `HUDMESSAGE_DONTSEND_WRAP` to lower bytes?

Reply:
I can try doing this. I might need to create new definitions for some of the flags, particularly `HUDMSG_ADDBLEND` and `HUDMSG_LOG` so they all don't occupy more than 16 bits. The four different HudMessage types will only ever occupy the first 2 bits, so this could work.